api/src/helpers/origin.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphQLError } from 'graphql';
+import wildcard from 'wildcard-match';
+
+export function isOriginValid(originParam: string): boolean {
+  if (originParam === '*') {
+    return true;
+  }
+  if (originParam.includes(' ')) {
+    return false;
+  }
```

Comment:
I believe it would be better for the user to trim every provided origin automatically instead of just invalidating it